lib/bolt_ext/puppetdb_inventory.rb

```diff
@@ -125,55 +149,83 @@ def build_parser
           opts.on('--key KEY', "Path to the private key") do |key|
             @cli_opts['key'] = key
           end
-          opts.on('--token-file TOKEN', "Path to the token file") do |token|
+          opts.on('--token-file TOKEN',
+                  "Path to the token file",
+                  "Default: #{Config::DEFAULT_TOKEN} if present") do |token|
             @cli_opts['token'] = token
           end
           opts.on('--url URL', "The URL of the PuppetDB server to connect to") do |url|
             @cli_opts['server_urls'] = [url]
           end
-          opts.on('--file FILE', "Path to the inventory file to use") do |file|
-            @inventory_file = File.expand_path(file)
+          opts.on('--configfile CONFIG',
```

Comment:
Naming of configfile and token-file feel inconsistent :(

Reply:
Based on puppetdb-cli and puppet-nogui, I believe this should be `--config`.

Reply:
I went for consistency with bolt (which uses `--configfile`) over consistency with client tools or good taste. The `--token-file` internal inconsistency seems like a problem though.

I'm fine with whatever, really.

Reply:
Maybe we should change bolt...

Reply:
Turns out `--config` is actually accepted by Bolt, because it's unambiguous.

Reply:
Changing Bolt seems like the best option. In that case, I'll go with `--config` and `--token-file` and we can deal with Bolt separately?

Reply:
Yeah, let's do that. It's not consistent with anything else we do. Even Puppet uses `--config`.